cloudbuild.yaml

```diff
@@ -90,6 +90,13 @@ steps:
     entrypoint: 'bash'
     args: ['./scripts/run_examples', '-s', 'base', '-l', 'cpp']
 
+  - name: 'gcr.io/oak-ci/oak:latest'
```

Comment:
I did not add a similar step for building `oak_abi`, since the core workspace depends on it. But perhaps it is better to have an explicit step for it, for easier debugging. WDYT?

Reply:
I don't think it has to be done for `oak_abi` seeing that it will be built by the server and examples. I just added this to ensure something tries to build the experimental crates so they don't rot.